react-task/src/components/Register.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+import ButtonComponent from "./ButtonComponent";
+import Select from "./Select";
+import Input from "./Input";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Register = () => {
+  const schools = [
+    "Hogwarts",
+    "Beauxbatons Academy of Magic",
+    "Castelobruxo",
+    "Durmstrang Institute",
+    "Ilvermorny",
+    "Mahoutokoro School of Magic",
+    "Uagadou School of Magic ",
+  ];
+
+  return (
+    <form class="container w-50 d-flex flex-column justify-content-center mt-5">
+      <h3 class="text-center mb-3">Register</h3>
+
+      <Input type="text" id="firstName" placeholder="First name"></Input>
```

Comment:
General standard is to self-enclose elements that don't have children elements

```suggestion
      <Input type="text" id="firstName" placeholder="First name" />
```